### PR TITLE
add validation-error.component.spec

### PR DESCRIPTION
**Pull request by mehmet-erim**

### CODE REVIEW

File: packages/core/src/tests/validation-error.component.spec.ts [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+      this.fixture = TestBed.createComponent(ValidationErrorComponent);
+      this.component = this.fixture.componentInstance;
+      this.fixture.detectChanges();
+    });
+
+    it('should be created', () => {
+      expect(this.component).not.toBeUndefined();
+    });
+
+    it('should be class equal to invalid-feedback', () => {
```
Comment (armanozak) [inline packages/core/src/tests/validation-error.component.spec.ts]: should have invalid-feedback as class

File: packages/core/src/tests/validation-error.component.spec.ts [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+      const el: HTMLElement = this.fixture.debugElement.query(By.css('div')).nativeElement;
+      expect(el.className).toEqual('invalid-feedback');
+    });
+
+    it('should be div elements length equal to errors length', () => {
```
Comment (armanozak) [inline packages/core/src/tests/validation-error.component.spec.ts]: should have as many div elements as errors

File: packages/core/src/tests/validation-error.component.spec.ts [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+      const elements = this.fixture.debugElement.queryAll(By.css('div'));
+      expect(elements.length).toEqual(this.errors.length);
```
Comment (armanozak) [inline packages/core/src/tests/validation-error.component.spec.ts]: `toBe` can be used here

File: packages/core/src/tests/validation-error.component.spec.ts [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+      this.fixture.detectChanges();
+    });
+
+    it('should be created', () => {
+      expect(this.component).not.toBeUndefined();
+    });
+
+    it('should be class equal to invalid-feedback', () => {
+      const el: HTMLElement = this.fixture.debugElement.query(By.css('div')).nativeElement;
+      expect(el.className).toEqual('invalid-feedback');
```
Comment (armanozak) [inline packages/core/src/tests/validation-error.component.spec.ts]: `toBe` can be used here

File: packages/core/src/tests/validation-error.component.spec.ts [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+    });
+
+    it('should be div elements length equal to errors length', () => {
+      const elements = this.fixture.debugElement.queryAll(By.css('div'));
+      expect(elements.length).toEqual(this.errors.length);
+    });
+
+    it('should have div element error message', () => {
```
Comment (armanozak) [inline packages/core/src/tests/validation-error.component.spec.ts]: should have error messages as text content of div elements

File: packages/core/src/tests/validation-error.component.spec.ts [hunk 6]
```diff
@@ -0,0 +1,65 @@
...
+      const element: HTMLElement = this.fixture.debugElement.query(By.css('div')).nativeElement;
+      expect(element.textContent).toEqual(this.errors[0].message);
```
Comment (armanozak) [inline packages/core/src/tests/validation-error.component.spec.ts]: `toBe` can be used here.

Also, there are two errors, so one could expect to see both error messages checked, right?

Comment (mehmet-erim) [reply packages/core/src/tests/validation-error.component.spec.ts]: Right, thanks for the suggestions. I updated.

Comment (mehmet-erim) [reply packages/core/src/tests/validation-error.component.spec.ts]: But I can't use `toBe` now.

**Pull request merged**